cmd/kubeadm/app/kubeadm.go

```diff
@@ -20,15 +20,19 @@ import (
 	"flag"
 	"os"
 
-	_ "github.com/golang/glog"
+	glog "github.com/golang/glog"
 	"github.com/spf13/pflag"
 
 	utilflag "k8s.io/apiserver/pkg/util/flag"
 	"k8s.io/kubernetes/cmd/kubeadm/app/cmd"
+	"k8s.io/kubernetes/pkg/version"
 )
 
 // Run creates and executes new kubeadm command
 func Run() error {
+	// To help debugging, immediately log version
+	glog.Infof("Version: %+v", version.Get())
```

Comment:
Why not using `kubeadm version` to get the version?

I don't think we should add it. Neither for above `Version: xx` changes.

Reply:
I remove it , THX